test/libsolidity/SolidityABIJSON.cpp

```diff
@@ -292,6 +308,8 @@ BOOST_AUTO_TEST_CASE(events)
 	{
 		"name": "f",
 		"constant": false,
+		"view": false,
```

Comment:
Enum of constant, view, pure?

Constant == view, it is just left in the ABI JSON for backwards compatibility.